2 seminar

Daria-Sinitsyna opened the pull request:
Исправила ошибку: не добавила интерфейс Serializable в Human и FamilyTree

Commit:
2 seminar

Liberate520 commented on src/seminar2/Main.java:
```diff
@@ -0,0 +1,69 @@
+
+    private static void save(FamilyTree tree, String path) throws IOException {
+        Writable file = new FileHandler();
+        file.save((Serializable) tree, path);
```
приведение типов не требуется. То есть надо удалить (Serializable)

Commit:
3 seminar

добавлены сортировки по имени, фамилии, возрасту, количеству детей в семье.

Добавлены связи супругов.

Добавлен метод выведения членов одной семьи (муж/жена/дети).

Commit:
add sort methods - Name/Surname/Age/Children
